src/test/java/com/ays/admin_user/controller/AdminUserControllerTest.java

```diff
@@ -0,0 +1,98 @@
+
+        // When
+        Page<AdminUserEntity> mockAdminUserEntities = new PageImpl<>(
+                AdminUserEntityBuilder.generateValidUserEntities(1)
+        );
+        List<AdminUser> mockAdminUsers = ADMIN_USER_ENTITY_TO_ADMIN_USER_MAPPER.map(mockAdminUserEntities.getContent());
+        AysPage<AdminUser> mockAysPageOfUsers = AysPage.of(mockAdminUserEntities, mockAdminUsers);
+        Mockito.when(adminUserService.getAdminUsers(mockAdminUserListRequest))
+                .thenReturn(mockAysPageOfUsers);
+
+        // Then
+        List<AdminUsersResponse> mockAdminUsersResponses = ADMIN_USER_TO_ADMIN_USER_RESPONSE_MAPPER.map(mockAysPageOfUsers.getContent());
+        AysPageResponse<AdminUsersResponse> pageOfAdminUsersResponse = AysPageResponse.<AdminUsersResponse>builder()
+                .of(mockAysPageOfUsers)
+                .content(mockAdminUsersResponses)
+                .build();
+        AysResponse<AysPageResponse<AdminUsersResponse>> mockAysResponse = AysResponse.successOf(pageOfAdminUsersResponse);
+        mockMvc.perform(AysMockMvcRequestBuilders
+                        .get(BASE_PATH, mockAdminUserToken.getAccessToken(), mockAdminUserListRequest))
+                .andDo(print())
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.time").isNotEmpty())
+                .andExpect(jsonPath("$.httpStatus").value(mockAysResponse.getHttpStatus().getReasonPhrase()))
+                .andExpect(jsonPath("$.isSuccess").value(mockAysResponse.getIsSuccess()))
+                .andExpect(jsonPath("$.response").isNotEmpty());
+
+        Mockito.verify(adminUserService, Mockito.times(1))
+                .getAdminUsers(mockAdminUserListRequest);
+    }
+
+    @Test
+    void givenValidUserListRequest_whenAdminUserUnauthorizedForListing_thenReturnAccessDeniedException() throws Exception {
+        // Given
+        AdminUserListRequest mockAdminUserListRequest = new AdminUserListRequestBuilder().withValidValues().build();
+
+        // Then
+        MockHttpServletRequestBuilder mockHttpServletRequestBuilder = AysMockMvcRequestBuilders
+                .get(BASE_PATH, mockUserToken.getAccessToken(), mockAdminUserListRequest);
+
+        AysResponse<AysError> mockResponse = AysResponseBuilder.UNAUTHORIZED;
+        mockMvc.perform(mockHttpServletRequestBuilder)
+                .andDo(MockMvcResultHandlers.print())
+                .andExpect(AysMockResultMatchersBuilders.status().isUnauthorized())
+                .andExpect(AysMockResultMatchersBuilders.time().isNotEmpty())
+                .andExpect(AysMockResultMatchersBuilders.httpStatus().value(mockResponse.getHttpStatus().name()))
+                .andExpect(AysMockResultMatchersBuilders.isSuccess().value(mockResponse.getIsSuccess()))
+                .andExpect(AysMockResultMatchersBuilders.response().doesNotExist());
+    }
+}
```

Comment:
In Java, adding a blank line at the end of a class file is recommended for better readability and adherence to coding conventions. It visually separates the class from other code and improves code organization. GitHub may display a warning if the blank line is missing to remind developers to follow this convention and maintain a consistent coding style.

Reply:
Ok. I added one line space at the end of the method and the end of a class file. Can you look through it again?

Reply:
@Rapter1990 No action has been taken regarding this comment. Can we review it again?